index.html

```diff
@@ -0,0 +1,99 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="project1.css">
+    <title>Main mobile</title>
+</head>
+
+<body>
+    <header>
+        <a class="welcome" href="">Welcome</a>
+        <nav class="menu">
+            <div class="hamburger"></div>
+            <div class="hamburger"></div>
+            <div class="hamburger"></div>
+        </nav>
+    </header>
+
+
+    <section class="professional">
```

Comment:
- [Optional] Please consider using an indentation of two spaces in all your documents for readability. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

- [Optional] Please make sure that each of your commit messages should be written in the imperative tense and capitalized. This will make your work look more professional. A good example of a professional commit message is *Add markup to index.html* and a bad example is *added HTML*. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8).